src/OrbitBase/include/OrbitBase/FutureHelpers.h

```diff
@@ -73,6 +73,82 @@ template <typename T>
   return promise->GetFuture();
 }
 
+template <typename T>
+[[nodiscard]] inline Future<ErrorMessageOr<T>> UnwrapFuture(
```

Comment:
does inline give us anything here?

Reply:
No, just a copy-and-paste artefact. I will remove it.

Reply:
Done.